scripts/instruction.js

```diff
@@ -0,0 +1,19 @@
+const modal = document.getElementById("modal__instruction");
+
+const btn = document.getElementById("btn__instruction");
+
+const span = document.getElementsByClassName("close")[0];
+
+btn.onclick = function() {
+  modal.style.display = "block";
+}
+
+span.onclick = function() {
+  modal.style.display = "none";
+}
+
+window.onclick = function(event) {
```

Comment:
we wszystkich trzech przypadkach zmień onclicki na addeventlistenery, Adam o tym ostatnio mówił :)